src/styles/blocks/main.scss

```diff
@@ -0,0 +1,32 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
The grid-template-columns property is set to repeat(auto-fill, 200px), but this may not be the best choice. Auto-fill tries to fill the row with as many columns as it can, which may not always produce the desired result. Instead, consider using auto-fit, which will create as many columns as possible, but will also expand those columns to fill any remaining space if there aren't enough items to fill them.